### ISSUE TITLE
Getting Clarity

### ISSUE THREAD

**Issue by Yehrman** [Compuskills-Jerusalem/Fitness-Project#25]
@ykosbie Hi all
I'm still struggling with getting clear what are the steps needed.I'd like to bounce this off all of you and If anyone can help clarify this it would be much appreciated.
The technical steps we're trying to take are to find a api meaning a pre built method that interacts with these devices fridge,pantry as well as maps and and then write code in c# to manipulate these api's to give real time warning
In short.
We are finding c# compatible fridge etc api's   
Sounds right?
